libs/etterlatte-ktor/src/testFixtures/kotlin/no/nav/etterlatte/ktor/BrukerTokenInfoTestGenerator.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.etterlatte.ktor
+
+import no.nav.etterlatte.libs.ktor.token.BrukerTokenInfo
+
+fun simpleSaksbehandler() = BrukerTokenInfo.of("token", "saksbehandler", null, null)
```

Comment:
Foreslår å slå saman denne og den under, og bruke "saksbehandler" som default-verdi for parameteren

Reply:
Agreed